artview/core/core.py

```diff
@@ -78,10 +78,14 @@ class Variable(QtCore.QObject):
         |VplotAxes  | Hold axes of      |:py:class:`matplotlib.axes.Axes`    |
         |           | Matplotlib figure |instance or None                    |
         +-----------+-------------------+------------------------------------+
-        |VPathInte\ | Hold auxiliar     |function like :py:func:`~artview.\  |
+        |VPathInte\ | Hold auxillary    |function like :py:func:`~artview.\  |
         |riorFunc   | function          |components.RadarDisplay.\           |
         |           |                   |getPathInteriorValues` or None      |
         +-----------+-------------------+------------------------------------+
+        |Vfilelist  | Hold filenames in |list containing string file         |
+        |           | current working   |basenames                           |
```

Comment:
The directory and filename are found from the ```Vradar.value.filename``` and then matched to a file in the filelist. If not found, it defaults back to the beginning of the directory. I don't think there is a way for this to happen, but it was a sanity check while I working on it and I just left it in just in case.